buildpacktoml_test.go

```diff
@@ -0,0 +1,729 @@
+package lifecycle_test
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle"
+	"github.com/buildpacks/lifecycle/launch"
+	"github.com/buildpacks/lifecycle/layers"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+	"github.com/buildpacks/lifecycle/testmock"
+)
+
+const latestBuildpackAPI = "0.5" // TODO: is there a good way to ensure this is kept up to date?
```

Comment:
Instead of defining this constant you could use `api.Buildpack` and pull the last element or a helper func to the `api` package to return the latest api (if you have conerns about relying on the order), as it seems like it could be a generally useful.